bench/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "sanctuary-type-classes-benchmarks",
+  "version": "0.0.0",
+  "private": true,
+  "license": "WTFPL",
```

Comment:
I guess we can change it to MIT, now that this "package" only contains code specific to this project. What do you prefer? npm wants us to provide a license.